_conferences/droidcon-lisbon-2024.md

```diff
@@ -0,0 +1,13 @@
+---
+name: "Droidcon"
+website: https://www.lisbon.droidcon.com/
+location: Lisbon, Portugal
+
+date_start: 2024-09-05
+date_end:   2024-09-06
+
+cfp:
+  start: 2024-04-01
```

Comment:
For me the sessionize page shows that the call for speakers opened 2024-02-01

![image](https://github.com/AndroidStudyGroup/conferences/assets/218061/cfe0a2ac-6988-4a69-8731-233b9d5437cc)

Reply:
Oh crap sorry i messed up the date there my bad fixing now